src/test/mir-opt/unusual-item-types.rs

```diff
@@ -1,12 +1,18 @@
 // Test that we don't ICE when trying to dump MIR for unusual item types and
 // that we don't create filenames containing `<` and `>`
+// compile-flags: --emit mir
```

Comment:
This shouldn't be needed

Reply:
Apparently we don't automatically generate this MIR. Could you add `let _ = Test::X as fn(usize) -> Test;` to `main` instead?

Reply:
:+1: Thanks for the suggestion. This is much cleaner.